.github/workflows/pr.yaml

```
@@ -288,43 +303,24 @@ jobs:
     - name: Install Planemo
       run: pip install planemo
     - name: Planemo ci_find_tools
-      run: |
-        touch changed_repositories_chunk.list changed_tools_chunk.list
-        if [ -s ../workflow_artifacts/changed_repositories.list ]; then
-            if [ $(wc -l < ../workflow_artifacts/changed_repositories.list) -eq 1 ]; then
-                planemo ci_find_tools --chunk_count ${{ needs.setup.outputs.nchunks }} --chunk ${{ matrix.chunk }} \
-                               --output changed_tools_chunk.list \
-                               $(cat ../workflow_artifacts/changed_repositories.list)
-            else
-                planemo ci_find_repos --chunk_count ${{ needs.setup.outputs.nchunks }} --chunk ${{ matrix.chunk }} \
-                               --output changed_repositories_chunk.list \
-                               $(cat ../workflow_artifacts/changed_repositories.list)
-            fi
-        fi
-    - name: Show changed tools/repositories chunk list
-      run: cat changed_tools_chunk.list changed_repositories_chunk.list
+      run: planemo ci_find_tools --chunk_count ${{ needs.setup.outputs.nchunks }} --chunk ${{ matrix.chunk }} --exclude packages --exclude deprecated --exclude_from .tt_skip --output changed_tools_chunk.list $(cat ../workflow_artifacts/changed_repositories.list)
```

We still need to check that `changed_repositories.list` is not empty:

```suggestion
      run: |
        touch changed_tools_chunk.list
        if [ -s ../workflow_artifacts/changed_repositories.list ]; then
          planemo ci_find_tools --chunk_count ${{ needs.setup.outputs.nchunks }} --chunk ${{ matrix.chunk }} --exclude packages --exclude deprecated --exclude_from .tt_skip --output changed_tools_chunk.list $(cat ../workflow_artifacts/changed_repositories.list)
        fi
```